docs/content/config/security/rspamd.md

```diff
@@ -97,6 +97,26 @@ You can also have comments (the line starts with `#`) and blank lines in `rspamd
 
     These simple commands are meant to give users the ability to _easily_ alter modules and their options. As a consequence, they are not powerful enough to enable multi-line adjustments. If you need to do something more complex, we advise to do that [manually](#manually)!
 
+## Advanced Configuration
+
+### _Abusix_ Integration
+
+This subsection gives information about the integration of [Abusix], "a set of blocklists that work as an additional email security layer for your existing mail environment". The setup is straight-forward and well documented:
+
+1. [Create an account](https://app.abusix.com/signup)
+2. Retrieve your API key
+3. Navigate to the ["Getting Started" documentation for Rspamd][abusix-rspamd-integration] and follow the steps described there
+4. Make sure to change `<APIKEY>` to your private API key
+
+We recommend mounting the files directly into the container, as they are rather big and not manageable with the [modules script](#with-the-help-of-a-custom-file). If mounted to the correct location, Rspamd will automatically pick them up.
+
+While _Abusix_ can be integrated into Postfix, Postscreen and a multitude of other software, we recommend integrating _Abusix_ only into a single piece of software running in your mail server - everything else would be excessive and wasting queries. Moreover, we recommend the integration into suitable filtering software and not Postfix itself, as software like Postscreen or Rspamd can properly evaluate the return codes and other configuration.
+
+[Abusix]: https://abusix.com/
+[abusix-rspamd-integration]: https://docs.abusix.com/abusix-mail-intelligence/gbG8EcJ3x3fSUv8cMZLiwA/getting-started/dmw9dcwSGSNQiLTssFAnBW#rspamd
+
+[//]: # (General Links)
```

Comment:
What is this?

Reply:
Didn't want to mess up the general Rspamd links with links coming from examples, hence the example links for _Abusix_, and then a comment that shows that the general links are following.

Reply:
> `[//]: # (General Links)`

This is meant to be a markdown comment?

```suggestion
<!-- General Links -->
```

Reply:
The one you proposed works for GitHub (when rendering it to HTML), mine should be the most compatible for other platforms though \[[source](https://stackoverflow.com/a/20885980)\]. I don't mind really, but I'm used to `[//]: # ()` and it always worked for me in the past. We could change it though..

Reply:
There are some extensions that we could use with [mkdocs to strip out comments](https://github.com/mkdocs/mkdocs/issues/1303). Or better having html processed instead to strip them probably :man_shrugging: 

Personally, I don't have an issue with them being present in the HTML markup, they're not rendered.

---

I appreciate the source link, thanks for that :+1: 

It still seems like a hacky workaround though. I'd like the CI to run something like `lychee` for docs PRs (_I had some WIP work a while back, but didn't get around to finishing it_) so that links are checked. Perhaps it won't get confused by that syntax :man_shrugging:

I'll approve as it's not causing any problems atm, my preference for the HTML comment syntax is it'd likely be more familiar to anyone that comes across the line, less confusion is good but I'm not going to be fussy here until someone else expresses the same confusion :sweat_smile:

Reply:
Alright 👍🏼